purchase_fiscal_position_update/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Purchase Fiscal Position Update
+===============================
+
+With this module, when a user changes the fiscal position of a purchase
+order, the taxes on all the purchase order lines are automatically
+updated.
+
+Configuration
+=============
+
+No specific configuration needed. This module uses the standard
+configuration of the fiscal positions.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/purchase-workflow/10.0
```

Comment:
You need to set repo_id:
:target: https://runbot.odoo-community.org/runbot/142/10.0